### PR TITLE
Update Doxyfile.template

### PR DESCRIPTION
**Pull request by maxm11**
Added the project name and document version to the Doxyfile.

### CODE REVIEW

**Comment by gunnarx** [comment]
1. I like the initiative but I think no one in our community knows the project as "Wayland InVehicle Infotainment Extension".   If it is known, it is known as Wayland IVI extension, because the name has been used so long.

2. Is there a special thought behind writing the version in reverse order (0.0.2 instead of 2.0.0)?

(I'll let the real maintainers take it from here...)

File: doc/Doxyfile.template [hunk 1]
```diff
@@ -26,13 +26,13 @@ DOXYFILE_ENCODING      = UTF-8
...
 
-PROJECT_NAME           = "___DOC_NAME___"
+PROJECT_NAME           = "Wayland InVehicle Infotainment Extension"
 
 # The PROJECT_NUMBER tag can be used to enter a project or revision number.
 # This could be handy for archiving the generated documentation or
 # if some version control system is used.
 
-PROJECT_NUMBER         = ___DOC_VERSION___
+PROJECT_NUMBER         = 0.0.2
```
Comment (efriedrich) [inline doc/Doxyfile.template]: it should be 2.2.0, in general it would make sence to rework the project number assingment and make it centralized at one place but i would prefer to do it in a separate commit...
so here just change to 2.2.0

**Comment by efriedrich** [comment]
@maxm11 would you like to continue to work on the change request? or should we just close and create a new one or force push changes to this one(not sure if is possible with current settings)?
